docs/schemas/resource/IfcPresentationAppearanceResource/Types/IfcFontWeight.md

```diff
@@ -3,7 +3,6 @@
 The _IfcFontWeight_ type defines the weight of the font. Values are:
 
 * normal
```

Comment:
```suggestion
* normal
* small-caps [DEPRECATED]
```

If we want correspondence with the formal rule we need to add `small-caps`. But if you don't mind I'll add a deprecated label next to it (we don't have that formally since this is just a string.